### PR TITLE
Add PackedVector4Array and Update Extension Api

### PR DESCRIPTION
**Pull request by ZerxZ**

### CODE REVIEW

**Comment by raulsntos** [comment]
Thanks for the contribution. It may take me a while to review this because I'm currently using Godot builds at commit godotengine/godot@652438a3952ae13e12dd4fd40756311fde81a791, to prevent unrelated changes to Godot breaking the bindings. I'll try to review as soon as I can, but I'm still finding many bugs that I'd like to fix first. I apologize for the inconvenience.

**Review by raulsntos — APPROVED** [review]

**Comment by raulsntos** [comment]
Thanks!

File: src/Godot.BindingsGenerator/BindingsGenerator/BindingsData.RegisterKnownTypes.cs [hunk 1]
```diff
@@ -216,6 +216,7 @@ private void RegisterKnownTypes()
         RegisterPackedArray("PackedVector2Array", KnownTypes.GodotVector2, KnownTypes.GodotPackedVector2Array, KnownTypes.NativeGodotPackedVector2Array);
         RegisterPackedArray("PackedVector3Array", KnownTypes.GodotVector3, KnownTypes.GodotPackedVector3Array, KnownTypes.NativeGodotPackedVector3Array);
         RegisterPackedArray("PackedColorArray", KnownTypes.GodotColor, KnownTypes.GodotPackedColorArray, KnownTypes.NativeGodotPackedColorArray);
+        RegisterPackedArray("PackedVector4Array", KnownTypes.GodotVector3, KnownTypes.GodotPackedVector4Array, KnownTypes.NativeGodotPackedVector4Array);
```
Comment (Delsin-Yu) [inline src/Godot.BindingsGenerator/BindingsGenerator/BindingsData.RegisterKnownTypes.cs:219]: `KnownTypes.GodotVector3` this doesn't sound correct. : /

Comment (raulsntos) [reply src/Godot.BindingsGenerator/BindingsGenerator/BindingsData.RegisterKnownTypes.cs:219]: Oops, looks like I messed this up on the rebase.

Comment (raulsntos) [reply src/Godot.BindingsGenerator/BindingsGenerator/BindingsData.RegisterKnownTypes.cs:219]: Fixed in https://github.com/raulsntos/godot-dotnet/commit/c015204a971e78d7be1ff3f74497a45cc4ffebfa

**Pull request merged**